CONTRIBUTING.md

```diff
@@ -10,45 +10,81 @@ it can be incorporated into the master branch.
 To help developing `mne-bids`, you will need a few adjustments to your
 installation as shown below.
 
-##### Running tests
+Running tests
+-------------
```

Comment:
this is a markdown file not an RST file. # should be used for titles

Reply:
Actually both is valid MD: https://www.markdownguide.org/basic-syntax/#alternate-syntax
But I can switch to `#` for consistency